### ISSUE TITLE
Angular8?

### ISSUE THREAD

**Issue by jgwinner** [slavede/ngx-paginate#24]
I'm getting errors while npm installing, for example:

`npm WARN @angular/http@7.2.15 requires a peer of @angular/core@7.2.15 but none is installed. You must install peer dependencies yourself.`

I checked `package.json`, and it does look like ngx-paginate needs version 7. 

Are there any plans to update ngx-paginate?

Thanks,

        == John ==

**Comment by slavede** [comment]
It's ok, not an error, just a warning. If you already have ng set up it's cool., it will work.

**Comment by jgwinner** [comment]
I can't install it. It requires Angular 7, ONLY, and we're on Angular 8. node_modules can't be generated.

So, I took it out :(

**Comment by slavede** [comment]
I mean, I am using ng8 in other project and can easily install it and use it. 

What you've got is an npm warning about missing angular core version, but you have angular core even newer.

It's not possible by NPM to prevent installation. 

You should have tried starting the app and you would see that's not a problem.

Check more here

https://nodejs.org/es/blog/npm/peer-dependencies/

**Comment by jgwinner** [comment]
I took your code at face value, that it required the older version and hadn't been tested.

As long as we're throwing around shoulds, well, if it works, then maybe _you_ should update your dependencies?

We aren't use the control, yet, so starting the app wouldn't have proven much.

**Comment by slavede** [comment]
What about those using 7.0? Wouldn't that be more distractful saying you have too old version of Angular. If you know your way around angular one would know "ok, i have newer version of angular, it's backwards compatible for 99% of stuff, I can still use it". but if you would see you need newer version, one would question it.

**Comment by jgwinner** [comment]
It's a good question; I'm not an expert, but yours was the only package that had that warning when I did the upgrade.  

I checked ngx-pagination and they use ^8.0

If people are still using 7, they probably have a package-lock I'd think, which would constrain the version.

If you do update your ng version, obviously I"d update your version when you update the dependency; if people need to downgrade to ng 7, they just downgrade your package.

        == John ==
